db/migrate/20161022185733_create_user_apis.rb

```diff
@@ -0,0 +1,15 @@
+class CreateUserApis < ActiveRecord::Migration[5.0]
+  def change
+    create_table :user_apis do |t|
+      t.integer :uid
```

Comment:
Why `uid` instead of `user_id`? Could make some things a little tricky in the future unless there's something I'm missing (totally possible).